README.rst

```diff
@@ -47,7 +47,8 @@ Usage
 
 .. code-block:: python
 
- import pytest
+  import pytest
+  import os
```

Comment:
I know this is sample code, but `os` doesn't appear to be used.

Reply:
Oops :) definitely making #152 a priority!